src/theme.js

```diff
@@ -36,7 +29,16 @@ export const theme = createMuiTheme({
       fontSize: '14px',
       lineHeight: '24px',
       letterSpacing: 0.1,
-      color: 'rgba(255, 255, 255, 0.6)',
+      color: colors.textColor,
+    },
+    h6: {
+      fontFamily: 'Roboto',
```

Comment:
How come you are adding `h6` to `theme.js` in this pull request, if we already have it in `master`?
https://github.com/DarthVanger/NeuralNotes/blob/master/src/theme.js#L34